.dagger/build/sdk.go

```diff
@@ -128,18 +129,22 @@ func (build *Builder) typescriptSDKContent(ctx context.Context) (*sdkContent, er
 }
 
 func (build *Builder) goSDKContent(ctx context.Context) (*sdkContent, error) {
-	base := dag.Container(dagger.ContainerOpts{Platform: build.platform}).
+	base := dag.Container().
```

Comment:
Oops, this also isn't entirely right. We should preserve the platform of the build image.